ComfortZoneApp/app/src/main/java/com/example/comfortzone/fragments/MapFragment.java

```diff
@@ -24,10 +26,12 @@
 
 import java.util.List;
 
-public class MapFragment extends Fragment implements OnMapReadyCallback {
+public class MapFragment extends Fragment implements OnMapReadyCallback, UpdateCityListCallback {
 
-    private MapView mvMap;
     public static final int MAX_CITIES = 20;
+    private final LatLng MAP_CENTER = new LatLng(39.8283, -98.5795);
```

Comment:
what is this coordinate? center of United States? is there a way to auto move the camera that contains all the pins present instead?